index.html

```diff
@@ -0,0 +1,171 @@
+                        <li><a href="#">News</a></li>
+                        <button class="CC">CC Campain</button>
+                    </ul>
+                </div>
+
+                <div class="humburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            <!-- </div> -->
+        </nav>
+    </header>
+    <main class="main">
+        <div class="main-items">
+            
+            <div class="container">
+                
+                    <h1>"Hello!Sharing world"</h1>
+                    <h2>CREATIVE COMMONS GLOBAL SOMMIT 2015</h2>
+
+                    <div class="first-pragraph">
+                        <p>A joyful celebration believing in the value of openness and sharing, creating a positive
+                            change
+                            with
+                            people from over 80 countries is taking place in October, in Korea.</p>
+                    </div>
+                    <h3>2015.10.15(THU) ~ 16(FRI)</h3>
+                    <h4>@ National Museum of Karea, Art Center Nabi and more</h4>
+                
+            </div>
+        </div>
+        <div class="main-program">
+            <div class="container">
+            <h1>Main Program</h1>
+            <img src="images/title_underline.svg" alt="underline">
+            <div class="prog-sec">
+                <div class="prog-items">
+                    <img src="images/Lecture.svg" alt="lecture">
+                    <h2>Lecture</h2>
+                    <p>Listen to the speakers from various contries about the messages of sharing and openning.</p>
+                </div>
+
+                <div class="prog-items">
+                    <img src="images/CC.svg" alt="CC Excibition">
+                    <h2>CC Exhibition</h2>
+                    <p>Appreciate various creations applying CC license of artists, organized from art center Nabi.</p>
+                </div>
+
+                <div class="prog-items">
+                    <img src="images/forum.svg" alt="Forum">
+                    <h2>Forum</h2>
+                    <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+                </div>
+
+                <div class="prog-items">
+                    <img src="images/workshop.svg" alt="Workshop">
+                    <h2>Workshop</h2>
+                    <p>Try creating your own work using open source license rather than just wathcing</p>
+                </div>
+
+                <div class="prog-items">
+                    <img src="images/ignite.svg" alt="CC Ignite">
+                    <h2>CC Ignite</h2>
+                    <p>get opportunities to network with CC affiliates around the world, also after the summit.</p>
+                </div>
+            </div>
+            <a>see the whole Program</a>
```

Comment:
- [ ] Kindly fix the following.

- Behance design.
<img width="776" alt="image" src="https://github.com/abdozayan12/capstone-project---Conference-page/assets/41163571/22d1291b-4345-4e63-a55e-cf5404d36d11">

- Your design
<img width="557" alt="image" src="https://github.com/abdozayan12/capstone-project---Conference-page/assets/41163571/2ed8ac87-5a0c-4830-bcdc-c586cc0c6403">

- 1. The spacing beneath `Main Program` should be reduced.
- 2. The border there, indicates the effect on hover, meaning the hover effect of the boxes should be that. Kindly note that on adding hover effect the boxes might tend to shift when the border is applied on hover, how to prevent this is to make the border transparent on not-hover, the have the color on hover something like
```
.box {
  border: 2px solid transparent;
}
.box:hover {
  boder: 2px solid #fff;
} or .box:hover {
      border-color: #fff;
}, 
```
Note: use the same pixel size for both.
alternatively, you can use `outline`.

- 3. Increase the spacing.
- 4. Reduce the font-weight and line-height.